.eslintignore

```diff
@@ -1,4 +1,5 @@
 src/App.js
 src/index.js
 src/reportWebVitals.js
-src/components/Calculator.js
\ No newline at end of file
+src/components/Calculator.js
+src/components/Home.js
```

Comment:
Kindly delete this file as it aims to defeat the purpose of having linters set up. Linters represent coding standards that you share with your team and attempts should not be made to bypass its checks.